### PR TITLE
user permission clarification

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#848

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ There are a few different ways you can access a mailbox once you've been given a
 
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.  
     
-2. Select the name of the user to open their properties pane.
+2. Select the name of the user (from which you plan to give a sending permission) to open their properties pane.
```
Comment (JohanFreelancer9) [inline Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
2. Select the name of the user (from whom you plan to give a sending permission) to open their properties pane.
```

File: Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ There are a few different ways you can access a mailbox once you've been given a
 
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.  
     
-2. Select the name of the user to open their properties pane.
+2. Select the name of the user (whose mailbox you want to allow to read) to open their properties pane.
```
Comment (JohanFreelancer9) [inline Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
2. Select the name of the user (whose mailbox you want to allow to be read) to open their properties pane.
```

File: Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md [hunk 3]
```diff
@@ -155,7 +155,7 @@ There are a few different ways you can access a mailbox once you've been given a
 
 1. In the admin center, go to the **Users** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834822" target="_blank">Active users</a> page.  
 
-2. Select the name of the user to open their properties pane.
+2. Select the name of the user (from which you plan to give a send on behalf permission) to open their properties pane.
```
Comment (JohanFreelancer9) [inline Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md]:
```suggestion
2. Select the name of the user (from whom you plan to give a **Send on behalf** permission) to open their properties pane.
```

**Comment by mypil** [comment]
@MaratMussabekov - please merge the commit on the copy/edit review done by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

File: Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md (hunk 2 above)
Comment (mypil) [reply Office365-Admin/add-users/give-mailbox-permissions-to-another-user.md]: @MaratMussabekov - kindly merge this copy/edit commit as well. thanks.

**Comment by mypil** [comment]
@kwekuako - PR has been copy edited and is ready for final review. This is based on issue #848 

Thank you.

cc: @kenwith

**Pull request merged**